nengo_bones/templates/LICENSE.rst.template

```diff
@@ -6,27 +6,30 @@
 
 Copyright (c) {{ copyright_start }}-{{ copyright_end }} {{ author }}
 
-{% if type == "nengo" %}
-{{ project_name }} is made available under a proprietary license
-that permits using, copying, sharing, and making derivative works from
-{{ project_name }} and its source code for any non-commercial purpose,
-as long as the above copyright notice and this permission notice
-are included in all copies or substantial portions of the software.
-
-If you would like to use {{ project_name }} commercially,
-licenses can be purchased from {{ author|rstrip('.') }}.
-Please contact {{ author_email }} for more information.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
-{% elif type == "proprietary" %}
+{% if license.startswith("abr") %}
+**ABR License**
+
+{{ project_name }} is made available under a proprietary license, the
+"ABR TECHNOLOGY LICENSE AND USE AGREEMENT" (the "ABR License").
+The main ABR License file is available for download at
+`<https://www.appliedbrainresearch.com/license>`_.
+The entire contents of this ``LICENSE.rst`` file, including any
+terms and conditions herein, form part of the ABR License.
+
+Commercial Use Licenses are available to purchase for a yearly fee.
+Academic and Personal Use Licenses for {{ project_name }} are available at
```

Comment:
Very minor/ignorable, but I'd probably just remove `for {{ project_name }}` (it's already mentioned in the title and previous paragraph).

Reply:
I put it there is as I worried that people would think all ABR licensed products have a no cost option; I didn't put it in the previous sentence because it's true for all products. The rest of the license is the same for all so I wanted to be extra clear that not all products have a no cost option, but for this project, there is one.